packages/cli/src/utils/checks.ts

```diff
@@ -168,6 +206,14 @@ class CheckBuilder {
       })
     )
 
+  isVoteSignerOrAccount = () =>
+    this.addCheck(
+      `${this.signer!} is vote signer or registered account`,
+      this.withAccounts(async (accs) => {
+        return accs.voteSignerToAccount(this.signer!).then(() => true, () => false)
```

Comment:
are we doing this `this.signer!` on the other checks? Looks like an easy way for a developer error..  since signer is an optional parameter on the `newChecksBuilder`

Reply:
yes, this pattern is already used in several places